packages/host/tests/helpers/index.gts

```diff
@@ -600,6 +613,44 @@ export class TestRealmAdapter implements RealmAdapter {
     }
   }
 
+  createJWT(claims: TokenClaims, expiration: string, secret: string) {
+    let nowInSeconds = Math.floor(Date.now() / 1000);
+    let expires = nowInSeconds + ms(expiration) / 1000;
+    let header = { alg: 'HS256', typ: 'JWT' };
```

Comment:
let's name our algorithm for this jwt something more descriptive like "clear" or "testing"

Reply:
Looks like we can use "alg": "none" per https://datatracker.ietf.org/doc/html/rfc7519#section-6. We should make sure that our server implementation limits what algorithm is accepts.

Reply:
fixed